demosplan/DemosPlanCoreBundle/Logic/ContentService.php

```diff
@@ -68,7 +68,7 @@ public function getContentList(User $user, $limit = null): array
         // Legacy Arrays
         // @improve T13447
         $result = array_map($this->convertToLegacy(...), $globalContentEntries);
-        $sorted = $this->manualListSorter->orderByManualListSort('global:news', 'global', 'content:news', $result);
+        $sorted = $this->manualListSorter->orderByManualListSort('global:news', 'global', 'content:news', $result, $this->customerService->getCurrentCustomer());
```

Comment:
here you should explicit write the param customer: , because you don't set the other default value of orderByManualListSort.

Reply:
done 78b60c400b84d6a11c38c6c33343ebc10c37fd9b